### PR TITLE
Fixes issue #612 - Create new group named "profile"

### PR DESCRIPTION
**Pull request by KSchopmeyer**
This creates a new pywbemcli command group "profile" and moves the
profile oriented commands to that group.  The initial work is simply to
move the existing commands definitions for the server profiles and
server centralinsts commands from the server group to the profile group.

Note that it does NOT change any of the options of the command that lists the profiles or of the centralinsts command. This leave
Andy's question about changing the option --profile on the profile list commad unchanged rather than making it an argument.  Note that it might also be logical to change the name of the profile to --name or --profile-name.

**DISCUSSION**

### CODE REVIEW

**Comment by andy-maier** [comment]
Please rebase on latest master.

**Comment by andy-maier** [comment]
I'm ok with the direction, so please rerun this PR (e.g. rebase or amend).

File: pywbemtools/pywbemcli/_cmd_profile.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+
+@cli.group('profile', cls=PywbemcliGroup, options_metavar=GENERAL_OPTS_TXT,
+           subcommand_metavar=SUBCMD_HELP_TXT)
+@add_options(help_option)
+def profile_group():
+    """
+    Command group for WBEM server management profiles.
+
+    This command group defines commands to inspect and manage the WBEM
+    management registered profiles maintained by the WBEM server.
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_profile.py:51]: Let's consistently use the term "WBEM management profile"

File: pywbemtools/pywbemcli/_cmd_profile.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+              required=False,
+              help='Filter by the defined organization. (ex. -o DMTF')
+@click.option('-p', '--profile', type=str, metavar='PROFILE-NAME',
+              required=False,
+              help='Filter by the profile name. (ex. -p Array')
+@add_options(help_option)
+@click.pass_obj
+def profile_list(context, **options):
+    """
+    List management profiles advertised by the server.
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_profile.py:72]: Let's consistently use the term "WBEM management profile"

File: pywbemtools/pywbemcli/_cmd_profile.py [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+    Retrieve the CIM instances representing the WBEM management profiles
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_profile.py:74]: We should not say that it retrieves the CIM instances, instead let's talk about "listing the WBEM management profiles"

File: pywbemtools/pywbemcli/_cmd_profile.py [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+@click.option('--rd', '--reference-direction', 'reference_direction',
+              type=click.Choice(['snia', 'dmtf']),
+              default='dmtf',
+              show_default=True,
+              help='Navigation direction for association.')
+@add_options(help_option)
+@click.pass_obj
+def centralinsts(context, **options):
+    """
+    List central instances of mgmt profiles on the server.
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_profile.py:118]: Let's consistently use the term "WBEM management profile"

File: pywbemtools/pywbemcli/_cmd_profile.py [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+
+    Retrieve the CIM instances that are central instances of the specified
+    WBEM management profiles, and display these instances. By default, all
+    management profiles advertized on the server are used. The profiles
```
Comment (andy-maier) [inline pywbemtools/pywbemcli/_cmd_profile.py:122]: Change "are used" to something else, e.g. "are included".

File: docs/pywbemcli/commands.rst [hunk 6]
```diff
@@ -999,7 +999,98 @@ namespace as a table:
 
 See :ref:`pywbemcli qualifier enumerate --help` for the exact help output of the command.
 
-.. index:: pair: command groups; server commands
+.. index:: pair: command groups; profile commands
+
+.. index:: pair: command groups; profile commands
```
[the remark this answers is not in the document]
Comment (KSchopmeyer) [reply docs/pywbemcli/commands.rst:1004]: DONE

File: docs/changes.rst [hunk 7]
```diff
@@ -16,6 +16,11 @@ Released: not yet
 
 **Incompatible changes:**
 
+* Moving the commands "server profiles" and "serve centralinstsn" to the
```
[the remark this answers is not in the document]
Comment (KSchopmeyer) [reply docs/changes.rst:19]: DONE

File: pywbemtools/pywbemcli/_cmd_profile.py [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+# the corresponding option name, ex. 'include_qualifiers'. It should be
+# defined with underscore and not dash
+
+
+@cli.group('profile', cls=PywbemcliGroup, options_metavar=GENERAL_OPTS_TXT,
+           subcommand_metavar=SUBCMD_HELP_TXT)
+@add_options(help_option)
+def profile_group():
+    """
+    Command group for WBEM server management profiles.
```
[the remark this answers is not in the document]
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_profile.py:48]: DONE

File: pywbemtools/pywbemcli/_cmd_profile.py (hunk 1 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_profile.py:51]: DONE

File: pywbemtools/pywbemcli/_cmd_profile.py (hunk 3 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_profile.py:74]: DONE

File: pywbemtools/pywbemcli/_cmd_profile.py (hunk 4 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_profile.py:118]: DONE

File: pywbemtools/pywbemcli/_cmd_profile.py (hunk 5 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_profile.py:122]: DONE

**Comment by KSchopmeyer** [comment]
Fixed documented issues.  I also added  more data to the table title for the profiles list.

File: pywbemtools/pywbemcli/_cmd_profile.py (hunk 2 above)
Comment (KSchopmeyer) [reply pywbemtools/pywbemcli/_cmd_profile.py]: DONE

**Pull request merged**